### ISSUE TITLE
unable to install via container

### ISSUE THREAD

**Issue by viveksoundrapandi** [aquasecurity/kube-bench#178]
I tried installing kube bench via contianer approach. It said kube-bench is successfully installed. But running the ./kube-bench says unable to execute binary file
Attached screenshot 

[https://imgur.com/a/B6rCLrh](https://imgur.com/a/B6rCLrh)

**Comment by ttousai** [comment]
@viveksoundrapandi from the error message it looks like the kube-bench binary was not built or does is not executable. Please run the command again and share the output of running `ls` in your working directory.

```
$ docker run --rm -v `pwd`:/host aquasec/kube-bench:latest install
Unable to find image 'aquasec/kube-bench:latest' locally
latest: Pulling from aquasec/kube-bench
c67f3896b22c: Pull complete 
703edffa3688: Pull complete 
ed4395ca6255: Pull complete 
217f0c419495: Pull complete 
eb407a8e1322: Pull complete 
5d1e02169609: Pull complete 
Digest: sha256:26cb9ab200d26b1aea65874b0068806ca040f925abfa502f698753adc476184d
Status: Downloaded newer image for aquasec/kube-bench:latest
===============================================
kube-bench is now installed on your host       
Run ./kube-bench to perform a security check   
===============================================
$ ls -l
total 8560
drwxr-xr-x 6 root root    4096 Nov 17 16:34 cfg
-rwxr-xr-x 1 root root 8759077 Nov 17 16:34 kube-bench
```

**Comment by lizrice** [comment]
@viveksoundrapandi it sounds like the executable is not built for your architecture (we only build for x86-64 Linux at the moment). Are you on something like an ARM platform maybe?

**Comment by ttousai** [comment]
@viveksoundrapandi any progress on this?

**Comment by spuder** [comment]
Also ran into this when trying to run kube-bench on OSX. Would be good to clarify that this is linux only.

**Comment by ttousai** [comment]
I think we should remove the install from container section altogether. There are already a number of ways to run the tool and I think this section is not necessary.

**Comment by lizrice** [comment]
Docs fixed with #200
